packages/client/src/comp/Initialize.tsx

```diff
@@ -30,6 +37,45 @@ export const Initialize: React.FC = () => {
     })()
   }, [oid, setInitialData])
 
+  React.useEffect(() => {
+    // Google translation has some issues translating iframe embeds, we
+    // also store if users have clicked on this message using local storage
+    // which can also throw errors at embeds.
+    if (isIframeEmbedded()) {
+      return
+    }
+
+    // Only show this message if users are visiting the page for the first
+    // time, or if they have previously clicked on the translate button
+    // (If they ignore the toast the message is not shown again)
+    const ignoreTranslationToast = localStorage.getItem('ignoreTranslationToast')
+    if (ignoreTranslationToast) {
+      return
+    }
+
+    // Get only first two letters from language code, ignoring additional
+    // info like '-GB' in 'en-GB'
+    const langCode = navigator.language.substr(0, 2)
+    const langName = getLanguageName(langCode)
+    if (langName && langName !== 'English') {
```

Comment:
I really liked how you had this translated into multiple languages before.  Perhaps use those languages if it's one of the ones you had earlier and just the English if it's not?